api/documentation/controller.py

```diff
@@ -454,9 +454,35 @@ def addSchemes(self):
             }
         )
         self.addComponent(
-            'schema', 'BulkCirculation', 'list',
+            'schema', 'BulkCirculationEvents', 'array',
             {'items': 'string'}
         )
+        self.addComponent(
+            'schema', 'CustomListCollectionArray', 'array',
+            {
+                'array': {
+                    'type': 'array',
+                    'items': {'$ref': '#/components/schemas/CustomListCollection'}
+                }
```

Comment:
This can be simplified down to just `items: {'$ref': ...}`, right now it's causing some odd display issues.